ui/components/component-library/modal-overlay/modal-overlay.tsx

```diff
@@ -21,6 +21,7 @@ export const ModalOverlay: React.FC<ModalOverlayProps> = ({
     width={BLOCK_SIZES.FULL}
     height={BLOCK_SIZES.FULL}
     onClick={onClick}
+    aria-hidden="true"
```

Comment:
Improves accessibility for screen readers
<img width="1440" alt="Screenshot 2023-05-06 at 10 00 51 AM" src="https://user-images.githubusercontent.com/8112138/236588695-62b9c50f-1306-4914-b036-271cd2cbd8cd.png">